### PR TITLE
[AutoRelease] t2-communication-2021-03-29

### PR DESCRIPTION
**Pull request by msyyc**
https://github.com/Azure/sdk-release-request/issues/1306
--

### CODE REVIEW

File: sdk/communication/azure-mgmt-communication/setup.py [hunk 1]
```diff
@@ -82,7 +82,7 @@
     ]),
     install_requires=[
         'msrest>=0.5.0',
-        'msrestazure>=0.4.32,<2.0.0',
+        'azure-mgmt-core>=1.2.0,<2.0.0',
```
Comment (jsntcy) [inline sdk/communication/azure-mgmt-communication/setup.py:85]: Is this auto generated or manual updated?

Remove duplicated  `'azure-mgmt-core>=1.2.0,<2.0.0',`
just like below,

```
install_requires=[
        'msrest>=0.5.0',
        'azure-common~=1.1',
        'azure-mgmt-core>=1.2.0,<2.0.0'
    ]

```

File: sdk/communication/azure-mgmt-communication/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
 # Release History
 
+## 1.0.0b5 (2021-03-29)
```
Comment (jsntcy) [inline sdk/communication/azure-mgmt-communication/CHANGELOG.md:3]: We should release GA version `1.0.0`

File: sdk/communication/azure-mgmt-communication/setup.py (hunk 1 above)
Comment (msyyc) [reply sdk/communication/azure-mgmt-communication/setup.py]: duplicated 'azure-mgmt-core>=1.2.0,<2.0.0' should be removed. It is problem of auto-releaase. Another 'azure-mgmt-core'  is added by others according to commit history

File: sdk/communication/azure-mgmt-communication/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
+  - Model CommunicationServiceResource has a new parameter system_data
+  - Model Operation has a new parameter action_type
+  - Model Operation has a new parameter is_data_action
+  - Added operation CommunicationServiceOperations.check_name_availability
+
+**Breaking changes**
+
+  - Model Operation no longer has parameter properties
+
 ## 1.0.0b5 (Unreleased)
```
Comment (jsntcy) [inline sdk/communication/azure-mgmt-communication/CHANGELOG.md:16]:
```suggestion
```

**Review by jsntcy — APPROVED** [review]

**Pull request merged**